dashboard/db/migrate/20170222192220_remove_href_from_external.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveHrefFromExternal < ActiveRecord::Migration[5.0]
+  def up
+    External.all.each{|level| level.update!(properties: level.properties.except('href'))}
```

Comment:
Since there are only ~ 1600 of these, it's not a big deal, but for larger sets [find_each](http://apidock.com/rails/ActiveRecord/Batches/find_each) is preferred over `.all.each` No need to change this, just FYI